.azure-pipelines/daily-build.yml

```diff
@@ -96,6 +96,7 @@ jobs:
         Release/**/Az.*.private.dll
         Release/**/Microsoft.Azure.PowerShell.Authentication.dll
         Release/**/Microsoft.Azure.PowerShell.Authentication.ResourceManager.dll
+        Release/**/Microsoft.Azure.PowerShell.AuthenticationAssemblyLoadContext.dll
```

Comment:
Have you added this assembly into the task in our regular sign pipeline?

Reply:
Good point